.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -5,13 +5,37 @@ description: Create a report to help us improve
 body:
 - type: markdown
   attributes:
-    value: |
+    value: >
```

Comment:
I think we may want some blank lines between sections to make https://github.com/ansible/issue-forms-demo-for-ansible/issues/new?template=bug_report.yml a little more readable

Reply:
Then, this change is probably incomplete. `>` removes the line separators turning the neighboring lines into one. If there's one empty line, it'll preserve it. And for a paragraph separator, it'd require two empty lines.

@gundalow where exactly do you envision the need to separate lines? Could you put comments into specific places?

Reply:
Oh, I probably misunderstood the original comment. It looks like this is necessary indeed.